src/Bicep.Core.Samples/Files/baselines/InvalidResourceDerivedTypes_LF/main.formatted.bicep

```diff
@@ -13,9 +13,10 @@ type invalid10 = resource<'abc' 'def'>
 type invalid11 = resource<123>
 type invalid12 = resource<resourceGroup()>
 
-type thisIsWeird = resource</*
-*/'Astronomer.Astro/organizations@2023-08-01-preview'
-///  >
+type thisIsWeird = resource<
+  /*
+*/ 'Astronomer.Astro/organizations@2023-08-01-preview'
+  ///  >
```

Comment:
What's the expected behavior of the formatter here? The change is unexpected, but I'm not sure if the former or new behavior aligns with what we want to see.

Reply:
Discussed offline. The updated version is what the formatter should have been producing all along, but it will default to printing the unaltered input if the syntax caused a parse error to be raised.